LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/Views/SiteIssuesReporter/SiteIssuesReporter.swift

```diff
@@ -1,60 +0,0 @@
-//
-//  SiteIssuesReporter.swift
-//
-//  Copyright © 2024 DuckDuckGo. All rights reserved.
-//
-//  Licensed under the Apache License, Version 2.0 (the "License");
-//  you may not use this file except in compliance with the License.
-//  You may obtain a copy of the License at
-//
-//  http://www.apache.org/licenses/LICENSE-2.0
-//
-//  Unless required by applicable law or agreed to in writing, software
-//  distributed under the License is distributed on an "AS IS" BASIS,
-//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-//  See the License for the specific language governing permissions and
-//  limitations under the License.
-//
-
-import AppKit
-import Foundation
-import PixelKit
-import VPNPixels
-
-public struct SiteIssuesReporter {
```

Comment:
This was replaced with `ReportSiteIssuesPresenter`.  The naming was odd since it  didn't represent this class' responsibilities correctly..